opennms-webapp/src/main/webapp/WEB-INF/jsp/admin/notification/noticeWizard/eventNotices.jsp

```diff
@@ -109,31 +109,31 @@
           <c:forEach items="${notifications}" var="notification">
           <tr>
             <td>
-              <input type="button" class="btn btn-secondary" value="Edit" onclick="javascript:editNotice('${notification.escapedName}')"/>
+              <input type="button" class="btn btn-secondary" value="Edit" onclick="javascript:editNotice('${fn:escapeXml(notification.escapedName)}')"/>
```

Comment:
Escaping for JavaScript vs XML would be safer here. See https://github.com/OpenNMS/opennms/pull/1763/files#diff-22f9d3c942102d20d541afa9a9aa46ea3a31f641449162efa61ad774684f33e4R101